client/src/screens/client/Home/Home.js

```diff
@@ -50,7 +50,7 @@ export default function Home() {
           start: item.start_dt,
           end: item.end_dt,
           backgroundColor:
-            item.status && item.status == "D" ? "#ffab40" : "#2196f3",
+            item.status && item.status === "D" ? "#ffab40" : "#2196f3",
```

Comment:
Did you editor make this change? I have doubt with `===` here.

Reply:
Nope. I made this changes intentionally. My goal was to remove the warnings from front end server goals. React gives warning if we use == instead of ===. We must use strict type checking!